test/integration/apiserver.go

```diff
@@ -380,14 +378,14 @@ func instanceService(ctx context.Context, conn *grpc.ClientConn, tenantAccount *
 		offering := &catalogv1alpha1.Offering{
 			ObjectMeta: metav1.ObjectMeta{
 				Namespace: tenantAccount.Status.Namespace.Name,
-				Name:      strings.Join([]string{"dbs", serviceCluster.Name, providerAccount.Name}, "."),
+				Name:      strings.Join([]string{"dbs", "external", serviceCluster.Name, providerAccount.Name}, "."),
```

Comment:
same here

Reply:
We cannot use constants pkg in test, since constants are under `internal` package.

Reply:
that is sad

Reply:
hmm, if needed, we can move the whole constants package out of internal, what do you think?

Reply:
it's not a big deal, I'm ok to merge it as is, but yeah, I think constants should not be in the internals

Reply:
OK, let's do it in a dedicated PR and move the whole constants pkg out of internal

Reply:
👍